materials/outline.md

```diff
@@ -0,0 +1,38 @@
+# Entitlement Justice and Measures of Algorithmic Fairness
+
+## Introduction
+ - Summarize importance of algorithmic fairness measures
+ - Introduce the concept of entitlement justice
+ - Introduce argument
+   - Entitlement justice is important in algorithmic fairness, but historically neglected in discussions of fairness measures
+   - (Some/certain) counterfactuals measures emphasize entitlement justice, resolving this neglect
+
+## Entitlement Justice
+ - Present Nozick's theory of entitlement justice
+ - Aknowledge known issues, but extract key points of relevance
+ - Present advantages of entitlement justice (including examples from domains relevant to algorithmic decision makers)
+
+## Algorithmic Fairness Measures
+ - Present formalism of algorithmic fairness measures
+ - Present common measures and hisotry of their devlopment
+   - i.e, why was each measure developed, what does it aim to capture
+
+## Entitlement Justice in Algorithmic Fairness
+ - In what sense are algorithmic decision makers relevant to property decisions
```

Comment:
Currently relies on this idea that holdings have been earned prior to the algorithmic decision. I need to say a lot about this to support it.